DevEdu.API/DevEdu.DAL/Repositories/CourseRepository.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Dapper;
+using DevEdu.DAL.Models;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class CourseRepository : BaseRepository, ICourseRepository
+    {
+        public int AddCourse(CourseDto courseDto)
+        {
+            return _connection.QuerySingle<int>(
+                "dbo.Course_Insert",
```

Comment:
each name of stored procedures must be placed somewhere as a constant
otherwise, it's an example of the "magic numbers" antipattern